Calculator/Program.cs

```diff
@@ -11,20 +11,21 @@ class Program
         static void Main(string[] args)
         {
             bool exitLoop = true;
-            int caseChoice;
             IMathOperations mathOperations;
+
             do
             {                             
-                    Console.WriteLine("Choose operation:");
-                    Console.WriteLine("1. Add.");
-                    Console.WriteLine("2. Subtract.");
-                    Console.WriteLine("3. Multiply.");
-                    Console.WriteLine("4. Divide.");
-                    Console.WriteLine("0. Quit.\n");
-                try
+                Console.WriteLine("Choose operation:");
+                Console.WriteLine("1. Add.");
+                Console.WriteLine("2. Subtract.");
+                Console.WriteLine("3. Multiply.");
+                Console.WriteLine("4. Divide.");
+                Console.WriteLine("0. Quit.\n");
+                
+                if (Int32.TryParse(Console.ReadLine(), out int caseChoice))
```

Comment:
1. Use `int` instead of `Int32`.
2. Add variable for value returned by `TryParse` - too many things are happening in one line and it's hard to read.